docs/README.md

```diff
@@ -34,6 +34,16 @@
   - [routing](js/routing.md)
 - [Maintenance](maintenance)
   - [Dependency Update](maintenance/dependency.md)
+- [Convert React Boilerplate To Electron App](electron-version)
```

Comment:
The main readme terms of content shouldn't have 10 lines dedicated to the electron documentation.

Reply:
One line is enough. The main electron readme file has its own toc.